src/BloomBrowserUI/templates/bloom-foundation-mixins.pug

```diff
@@ -197,6 +197,23 @@ mixin editable(language)
 	//- .bloom-content1.bloom-editable(lang=language, contenteditable="true")&attributes(attributes)
 		block
 
+//- Putting "+clickableLink('href', boolean) here" should result in a clickable link with correct spacing.
+//- Use the boolean argument 'periodAfter' to insert a period directly behind the link.
+//- Putting "+clickableLink('href')#someId here" should give "<a id=someId ..."
+mixin clickableLink(href, periodAfter)
```

Comment:
Well, never mind. I guess pug doesn't see anything wrong with also using link() as a mixin. They have it as an example in the mixin docs:
```
mixin link(href, name)
  //- attributes == {class: "btn"}
  a(class!=attributes.class href=href)= name

+link('/foo', 'foo')(class="btn")
```